web/profiles/joinup/joinup.behat.inc

```diff
@@ -2659,4 +2667,39 @@ JS;
     Assert::assertNotEmpty($element);
   }
 
+  /**
+   * Creates the testing content blocks.
+   *
+   * @beforeScenario @contentBlock
+   */
+  public function createContentBlocks(): void {
+    $block = BlockContent::create([
+      'type' => 'simple_block',
+      'uuid' => 'ec092d17-ef18-42b0-b460-642871150cd3',
+      'status' => TRUE,
+      'info' => 'Legal notice',
+      'body' => [
+        'value' => 'Legal notice text.',
+        'format' => 'content_editor',
+      ],
+    ]);
+    $block->save();
+    $this->contentBlocks[$block->id()] = $block;
+    // As the page that contains the block is created before the block, we need
+    // a cache clear to account the presence of the new block.
+    drupal_flush_all_caches();
```

Comment:
Wow isn't this overkill? Can't we just clear the page cache of this page using the cache tag?

Having the whole cache unnecessarily cleared in the middle of a Behat tests slows things down by several seconds.

Reply:
Fixed. Clearing now only a cache item.